### PR TITLE
Fix prd2190

### PR DESCRIPTION
**Pull request by bogdando**
- Fixes PRD-2190

### CODE REVIEW

**Comment by bogdando** [comment]
Do not merge, under test for now

File: deployment/puppet/l23network/manifests/l3/ifconfig.pp [hunk 1]
```diff
@@ -295,6 +295,7 @@
     subscribe     => File["$interface_file"],
     refreshonly   => true,
   }
+  File<| title == "$interface_file" |> ->  L3_if_downup <||>
```
Comment (xenolog) [inline deployment/puppet/l23network/manifests/l3/ifconfig.pp]: -1.

it's broke quantum

Comment (bogdando) [inline deployment/puppet/l23network/manifests/l3/ifconfig.pp]: Can you elaborate please? How it might be possible?

**Comment by bogdando** [comment]
testing

**Comment by bogdando** [comment]
- no quantum regression, that is good.
- test failed, the patch fixes nothing for nova-network, that is bad :-(
  closed

**Pull request closed without merging**